### DISCUSSION TITLE
2024-08-19_최종 PR과 관련 질문

### DISCUSSION THREAD

**Discussion by tttghost** [tttghost/wheelmaker#42] [Q&A]
- 저번시간에 말씀주셨던 PR을 만들때 앞의 브랜치주소를 연계해서 만드는 방식으로 PR들을 만들었습니다.
- 이때 최종 PR만 Merge하게되면 나머지 PR들은 Merge하지 않아도 되는 것일까요?
- 쥐피티한테 물어본 결과 이렇게 대답하긴 하는데 보통 어떤 방식으로 하는지 궁금해 여쭙니다!

"최종 PR이 Merge되면 그 이전의 모든 PR들(각각의 브랜치들이 서로를 추적하고 있는 경우)이 자동으로 Merge된 것과 같은 효과를 얻게 됩니다. 따라서 중간 PR들을 별도로 Merge할 필요는 없습니다. 다만, 각 PR의 변경 사항이 정확히 원하는 대로 Merge되는지 확인하려면 PR을 단계별로 Merge하는 것도 고려해볼 수 있습니다."

**Comment by jongfeel** [comment]
나머지 pull reqeust 들도 merge가 진행 되어야 합니다.

**Reply by tttghost** [reply]
최종pr을 머지하고 앞에것들도 머지만 하면 될까요.?.?

**Reply by jongfeel** [reply]
혼자 작업하신 것도 있고 ckeck out 된 branch가 이전 branch를 바라보고 있으면
순서에 상관 없이 merge가 진행되면 됩니다.

**Comment by tttghost** [comment]
[like]  김지수(개발) reacted to your message:
________________________________
